project/source/game/Game.h

```diff
@@ -211,7 +211,7 @@ class Game final : public Engine
 	void ListQuadtreeNodes();
 
 	// profiler
-	int profiler_mode;
+	bool profiler_mode;
```

Comment:
po co ta zmiana? profiler ma 3 tryby (0)wyłączony, (1)profilowanie aktualizacji, (2)profilowanie renderowania.

Reply:
W takim razie czemu nie enum?

Reply:
bo to stare jest, wtedy jeszcze nie było enumów :P